summoner-cli/src/Summoner/Template/Cabal.hs

```diff
@@ -95,15 +94,42 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
           build-depends:       $settingsBaseType $baseBounds
                              $commaPreludeLibrary
 
-          ghc-options:         -Wall
-                               $ghcOptions
+          $ghcOptions
 
           default-language:    Haskell2010
     |] <> defaultExtensions
 
     baseBounds :: Text
     baseBounds = cabalBaseVersions settingsTestedVersions
 
+    ghcOptions :: Text
+    ghcOptions = case settingsGhcOptions of
+        [] -> defaultGhcOptions
+        x:xs ->
+            let customGhcOptions = T.intercalate "\n" $ x : map (T.replicate 21 " " <>) xs in
```

Comment:
Could `intercalateMap` function be handy in here?

Reply:
Yes, it can help here indeed! I missed that

Reply:
Or not.. looking into it I see, that the logic here is a bit different. Sorry, false alarm!